azext_iot/tests/central/json/scheduled_job.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "newScheduledJobId",
+    "displayName": "New Scheduled Job",
+    "description": "",
+    "group": "ad86219f-d224-48ad-a29b-b593a4e4bb38",
+    "data": [
+      {
+        "type": "command",
+        "target": "dtmi:modelDefinition:bhmvjf0oj:k6ulbcbyndp",
+        "path": "Reboot",
+        "value": null
+      }
+    ],
+    "schedule": {
+      "start": "2022-05-24T22:29:01Z",
+      "recurrence": "daily"
+    },
+    "enabled": true,
+    "completed": false,
+    "etag": "\"970070b9-0000-0800-0000-628d554b0000\""
```

Comment:
why nested "?

Reply:
Copied from API response. Will fix this.